Magento_Theme/web/js/theme.js

```diff
@@ -2,26 +2,21 @@ define([
     'jquery',
     'mage/smart-keyboard-handler',
     'mage/mage',
-    'mage/ie-class-fixer',
     'domReady!'
 ], function ($, keyboardHandler) {
     'use strict';
 
-    if ($('body').hasClass('checkout-cart-index')) {
-        if ($('#co-shipping-method-form .fieldset.rates').length > 0 &&
-            $('#co-shipping-method-form .fieldset.rates :checked').length === 0
-        ) {
-            $('#block-shipping').on('collapsiblecreate', function () {
-                $('#block-shipping').collapsible('forceActivate');
-            });
-        }
-    }
-
     $('.cart-summary').mage('sticky', {
         container: '#maincontent'
     });
 
     $('.panel.header > .header.links').clone().appendTo('#store\\.links');
+    $('#store\\.links li a').each(function () {
+        var id = $(this).attr('id');
 
+        if (id !== undefined) {
+            $(this).attr('id', id + '_mobile');
```

Comment:
Couldn’t figure out what this was for, but in theory it should work!

Reply:
Yeah, it's silly to bother with this code (outside of eliminating `mage/ie-class-fixer`) because it doesn't actually do anything for us! But on the off chance that someone else is using our fork...unlikely... this keeps Magento's intended functionality